tests/test_mask.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import pytest
+import geopandas as gpd
+import numpy as np
+
+sys.path.append('../modules/')
```

Comment:
The proper way to do this is to have the software be an installable package. We can help you with that, or you can look at example `setup.py` files from all of our other packages.

Reply:
Sounds good. I will have a look and update `requirements.txt` as I do it. Requirements are frequently changing for this project.